Changelog

```diff
@@ -1,3 +1,6 @@
+v0.5 :
```

Comment:
Please do not include version release information in your PR as you don't know how it will be named (could be 1.0 or 0.4.1). Prefere using `unreleased:` here, I will name it when releasing.

Reply:
well there's no problem! always you can override by a new commit by yourselft.. that's all